docs/en/overview.md

```diff
@@ -1 +1,73 @@
 # Overview
+
+This chapter introduces you to the framework of MMSegmentation, the basic conception of semantic segmentation, and provides links to detailed tutorials about MMSegmentation.
+
+## What is semantic segmentation?
+
+Semantic segmentation is the task of clustering parts of an image together that belong to the same object class.
+It is a form of pixel-level prediction because each pixel in an image is classified according to a category.
+Some example benchmarks for this task are Cityscapes, PASCAL VOC and ADE20K.
```

Comment:
Should we add links to these benchmarks?
Like [cityscapes](https://www.cityscapes-dataset.com/benchmarks/)